hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/KeyValueContainerCheck.java

```diff
@@ -186,8 +186,8 @@ private void checkContainerFile() throws IOException {
     }
 
     dbType = onDiskContainerData.getContainerDBType();
-    if (!dbType.equals(OZONE_METADATA_STORE_IMPL_ROCKSDB) &&
-        !dbType.equals(OZONE_METADATA_STORE_IMPL_LEVELDB)) {
+    if (!dbType.equals(CONTAINER_DB_TYPE_ROCKSDB) &&
```

Comment:
Thanks @hanishakoneru  for working on this. Not sure if I understand correctly, since we are removing LevelDB support on DN, do we still need to define two types of DB for container here?

Reply:
We are deprecating LevelDB support. I kept it there in case there are old containers using LevelDB.

I am up for removing LevelDB support completely. I think it is safe to assume that there would not be a Container using LevelDB.

Reply:
That makes sense to me.